README.md

```diff
@@ -1,47 +1,41 @@
-# Prismic Vue.js Example Blog
+# Prismic & VueJS Example Blog
```

Comment:
It's "Vue.js", not "VueJS". But after you've introduced it as "Vue.js", you can just call it "Vue" on subsequent references:

> Vue.js is a popular framework. If you're interested in Vue, he's how to get started.